### PR TITLE
updating share page and index

### PR DESCRIPTION
**Pull request by nylim11**

### CODE REVIEW

File: src/pages/index.hbs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: src/templates/layouts/landing.hbs
-title: An Invitation to Joy
+title: Ceaseless Prayer - An Invitation to Joy
```
Comment (chrislim) [inline src/pages/index.hbs:3]: Should we even have "An Invitation to Joy?"
Should we be consistent? "Personal prayer for everyone on earth?"
Something else?

File: src/pages/share.hbs [hunk 2]
```diff
@@ -17,12 +17,34 @@ headline: Share the joy of praying for others
 		<!-- Main -->
 			<section id="main" class="container">
 				<div class="box special">
-          <div class="infographic">
-            <span class="image fit"><img src="images/ceaseless_fb_infographic_en.jpg" alt="" /></span>
+          <h2><b>Help get the word out about Ceaseless!</b></h2>
+          <p>Below are some materials to help you share Ceaseless with your friends, family, church, and beyond.
```
Comment (chrislim) [inline src/pages/share.hbs]: Use the materials below to share Ceaseless with...

File: src/pages/share.hbs [hunk 3]
```diff
@@ -17,12 +17,34 @@ headline: Share the joy of praying for others
...
+            <br/>
+            Hover over an image and pick which social network you would like to share it with.</p>
+          <div class="share shortcuts">
+            <h2>Overview</h2>
+            <ul>
+              <li><a href="#infographic">Infographic</a></li>
+              <li><a href="#banner">Banner</a></li>
+              <li><a href="#ceaseless-info-video">Video</a></li>
```
Comment (chrislim) [inline src/pages/share.hbs]: A link to app cards they can print out too?

File: src/pages/share.hbs [hunk 4]
```diff
@@ -17,12 +17,34 @@ headline: Share the joy of praying for others
...
+          <br/>
+          <div id="banner">
+            <h2>Banner - Facebook/Twitter Profile Cover Image</h2>
+            <span class="image fit"><img class="shareable-class" src="images/ceaseless_android_lifestyle_shareable_banner.jpg" alt="Ceaseless Prayer - Pray for your world 3 friends at a time" /></span>
           </div>
           <br/>
-          <div class="video-container">
-            <iframe width="560" height="315" src="https://www.youtube.com/embed/1KUHsYbZz8k" frameborder="0" allowfullscreen></iframe>
+          <div>
+            <h2>Video - Personal Prayer for Everyone on Earth</h2>
```
Comment (chrislim) [inline src/pages/share.hbs:50]: We don't want to overwhelm the user, but should we link to the app store video and my talk video too?

File: src/templates/partials/social/addthis-toolbox.hbs [hunk 5]
```diff
@@ -1,2 +1,2 @@
 <!-- Go to www.addthis.com/dashboard to customize your tools -->
-<div class="addthis_sharing_toolbox" data-url="http://www.ceaselessprayer.com" data-title="An Invitation to Joy."></div>
+<div class="addthis_sharing_toolbox" data-url="http://www.ceaselessprayer.com?src=webshare" data-title="Check out this app that helps you pray for others!"></div>
```
Comment (chrislim) [inline src/templates/partials/social/addthis-toolbox.hbs:2]: Can you preview how the text looks in Facebook? Make sure it shows up nice and with a good cover image.

**Review by chrislim — CHANGES_REQUESTED** [review]
One more comment: is there a difference between the two images? They look the same...

**Comment by nylim11** [comment]
fyi - removed extra image

File: src/pages/share.hbs (hunk 3 above)
Comment (nylim11) [reply src/pages/share.hbs]: added

File: src/pages/share.hbs (hunk 2 above)
Comment (nylim11) [reply src/pages/share.hbs]: updated

File: src/pages/share.hbs [hunk 6]
```diff
@@ -17,12 +17,41 @@ headline: Share the joy of praying for others
 		<!-- Main -->
 			<section id="main" class="container">
 				<div class="box special">
```
Comment (chrislim) [inline src/pages/share.hbs:19]: does this all look good on mobile too? (shrink width of browser)

File: src/pages/index.hbs (hunk 1 above)
Comment (chrislim) [reply src/pages/index.hbs:3]: Did you not want to change this? By the way, this is for the landing page too since it's a template.

**Review by chrislim — APPROVED** [review]
If my last comments are addressed you can ship it.

File: src/templates/partials/social/addthis-toolbox.hbs (hunk 5 above)
Comment (nylim11) [reply src/templates/partials/social/addthis-toolbox.hbs:2]: can't test this til it's shipped

**Pull request merged**